units/github_status_api.go

```diff
@@ -46,7 +46,7 @@ const (
 	noChildPatchTasksOrVariants = "no tasks/variants were configured for child patch"
 	NoSyncTasksOrVariants       = "no tasks/variants were configured for sync"
 	GitHubInternalError         = "GitHub returned an error"
-	InvalidConfig               = "config file was invalid"
+	InvalidConfig               = "config file was invalid: please sync with base branch and run `evergreen validate`"
```

Comment:
Is it possible for project level settings to also throw errors? In which case I think the guidance should be `evergreen validate -p <project>`